src/client/store/ui/userManagement/slice.ts

```diff
@@ -24,6 +28,12 @@ export const userManagementSlice = createSlice({
     setUserToEdit: (state, { payload }) => {
       state.user = payload
     },
+    updateCountryFilter: (state, { payload }) => {
+      state.filters.countries = payload
+    },
+    updateRolesFilter: (state, { payload }) => {
+      state.filters.roles = payload
+    },
```

Comment:
ideally one action is needed, that takes a parameter which is a Partial of UserFilter